core/spec/models/spree/store_spec.rb

```diff
@@ -1,24 +1,48 @@
 require 'spec_helper'
 
 describe Spree::Store, :type => :model do
+  # there is a default store created with the test_app rake task.
+  let(:store_1) { Spree::Store.first || create(:store) }
```

Comment:
I think we should replace Spree::Store.first with Spree::Store.where(default: true).first